benchmarks/sequel/benchmark.rb

```diff
@@ -0,0 +1,41 @@
+require "harness"
+
+Dir.chdir __dir__
+use_gemfile
+
+require "sequel"
+
+# Base Sequel, no plugins
+DB = Sequel.sqlite
+
+DB.create_table :posts do
+  primary_key :id
+  String :title, null: false
+  String :body
+  String :type_name, null: false
+  String :key, null: false
+  Integer :upvotes, null: false
+  Integer :author_id, null: false
+  DateTime :created_at, default: Sequel::CURRENT_TIMESTAMP
+  DateTime :updated_at, default: Sequel::CURRENT_TIMESTAMP
+end
+
+class Post < Sequel::Model; end
+
+50000.times {
```

Comment:
I copied the same test setup as the ActiveRecord benchmark for consistency but here's some data:

```
     Seed    Total
50K  19.0s   29.2s
10K   3.9s   14.4s
 1K   0.4s   10.8s
```

I can reduce it to 10K or some other number if you'd like??? Should we do the same for AR benchmark too?

Reply:
I reduced it to 10K seeds

Reply:
Thanks, I think that's better (1k would be fine too probably and save 4s more).
A 19s setup for the 50k seems pretty slow if not necessary.
It might be nice to update the activerecord benchmark too (e.g. so timings can be compared), not sure why it uses 50k record and only queries the first 1000.
@maximecb What do you think? I found this discussion about the AR benchmark: https://github.com/Shopify/yjit-bench/pull/5#discussion_r617030574